openvdb/python/pyGrid.h

```diff
@@ -1195,6 +1198,25 @@ copyToArray(GridType& grid, py::object arrayObj, py::object coordObj)
     op();
 }
 
+
+template<>
+inline void
+copyFromArray(points::PointDataGrid& grid, py::object arrayObj, py::object coordObj, py::object toleranceObj)
```

Comment:
Change this line to
`copyFromArray(points::PointDataGrid&, py::object /*arrayObj*/, py::object /*coordObj*/, py::object /*toleranceObj*/)`
to avoid unused parameter compiler warnings.

This change does indeed resolve the deleted function errors, though!